extensions/hibernate-orm/runtime/pom.xml

```diff
@@ -36,6 +36,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-narayana-jta</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-graalvm</artifactId>
+        </dependency>
```

Comment:
This needs to be removed since it's a duplicate of the other `hibernate-graalvm` dependency.

Reply:
thanks for spotting that! Done